src/lib.rs

```diff
@@ -135,6 +135,7 @@ pub mod dev {
     pub use crate::info::ConnectionInfo;
     pub use crate::rmap::ResourceMap;
     pub use crate::service::{HttpServiceFactory, ServiceRequest, ServiceResponse, WebService};
+    pub use actix_service::fn_factory;
```

Comment:
`fn_factory` docs use the `fn_service` function which looks convenient but is not re-exported. Shouldn't it be re-exported too?

Reply:
It's not really useful because it serves the same purpose as a hander.

Reply:
I mean that it's used in the example of this function's documentation and it **looks** convenient, so some folks who'll bump into it may want to use it according to the provided example, only to find out that it's not available through `actix-web`.

Like, it's not about actual helpfulness, but rather about avoiding a little point of frustration.

Reply:
I see your point, I'll let @robjtede chime in on the import situtation here and with the `dev` module.

Reply:
I'm (barely) on the side of exporting `fn_service`. Also re-exporting the `always_ready!` and `forward_ready!` macros would help here and in other places.